### ISSUE TITLE
Passat GTE NOFPA only GTE Mode RS4 Sound

### ISSUE THREAD

**Issue by Heeler4uu** [jilleb/mqb-soundaktor#31]
Hi can you make a file for Passat GTE without FPA and the FL_8W0907159D__0006_UNLOCKED.odx so that the Sound is only in GTE Mode. I try to modify the FPA Switch from 7 tor 3 in the Gateway Dataset, but i get incorrect Dataset Error. Actually i Drive with the NOFPA Version from the 8W0907159D, this is very Cool, but in E-Mode and the other Modes too much.

**Comment by Mjukes** [comment]
This is what I am after as well. I’d be happy to make a donation for it. 
 Needs to be loud so maybe a couple of volume variants if possible?
I do have FPA but I’d only want it in GTE mode. 

Again, happy to pay.

**Comment by capisci2000** [comment]
Hi at all, Does anyone have any news on this?

**Comment by capisci2000** [comment]
Hi everyone, I wanted to ask if it is possible to load the RS5 firmware onto a Passat GTE without driving profiles, but only with GTE mode. It is possible to modify the file in order to have the RS5 sound only in GTE mode. I think A3 E-tron is similar
Thank you

**Comment by Mjukes** [comment]
I have exactly this.  I paid Jille to make me a version. I didn't want it too loud,  just subtle but better than the silence I had before.

**Comment by capisci2000** [comment]
thanks for the reply, how did you contact him directly? Can I ask you how you did the flashing of the control unit step by step? do you have vcp or odis? Isn't it possible to do this with obdeleven? Can you confirm that the sound is increased only in gte mode and instead remains unchanged in the other modes: hybrid and E-mode? do you also have a passat gte without drive select profiles? Thank you

**Comment by Mjukes** [comment]
I emailed him (it's on here somewhere).  I have a GTE with driver profiles but they are all silent except GTE mode.  You cannot do it with OBDeleven,  you'll need a copy of ODIS or VCDS.

**Comment by capisci2000** [comment]
Hi ok, I'll try to find his email to contact him, did you personally flash your control unit? did you use vcds? I have vcds as well as obdeleven and can you confirm that it can also be done with vcds? you also remember how much cost the firmware to load in ECU?
Thanks

**Comment by Mjukes** [comment]
I use ODIS. Very easy to flash.

**Comment by capisci2000** [comment]
Can I have your contact to share some info about flash?
Thanks

**Comment by Mjukes** [comment]
Sure

Mike

At

Mookistar

D0t c0m

**Comment by Lazzor313** [comment]
Hey @Mjukes , can you confirm that I understand you correctly:

A GTE without FPA and a custom Soundaktor firmware (one that does not need FPA) will infact activate the soundaktor even when not in GTE mode?

I am trying to figure out this out before I start messing around with Hex coding. 

Thanks in advance!

**Comment by Mjukes** [comment]
> Hey [@Mjukes](https://github.com/Mjukes) , can you confirm that I understand you correctly:
> 
> A GTE without FPA and a custom Soundaktor firmware (one that does not need FPA) will infact activate the soundaktor even when not in GTE mode?
> 
> I am trying to figure out this out before I start messing around with Hex coding.
> 
> Thanks in advance!

This I can't tell you.  I have a file with Soundaktor ONLY in GTE mode so in theory a non-FPA car should be silent when not in GTE.

**Comment by Lazzor313** [comment]
> > Hey [@Mjukes](https://github.com/Mjukes) , can you confirm that I understand you correctly:
> > A GTE without FPA and a custom Soundaktor firmware (one that does not need FPA) will infact activate the soundaktor even when not in GTE mode?
> > I am trying to figure out this out before I start messing around with Hex coding.
> > Thanks in advance!
> 
> This I can't tell you. I have a file with Soundaktor ONLY in GTE mode so in theory a non-FPA car should be silent when not in GTE.

Yeah that would make sense, would you mind sharing the file so I can check out the values in the 010 editor? I know you paid for it, so it is cool if you don't want to share.

**Comment by Mjukes** [comment]
Yeah sorry,  I wouldn't be able to share it for free,  sorry.

**Comment by Lazzor313** [comment]
All good, I'll try to wing it.

**Comment by Mjukes** [comment]
Good luck!!
